src/main/openapi/openapi_v1.yaml

```diff
@@ -137,6 +137,18 @@ paths:
             example: 10
             default: 10
 
+        - name: start
+          in: query
+          description: | 
+            For pagination. Only return documents in the resultset from position start and forward.                        
+          required: false
+          schema:
+            type: integer
+            format: int32
+            example: 20
```

Comment:
In principle this is a fine example value, but unless the user actively changes it, it will be used by the OpenAPI GUI. I suggest setting it to 0 to avoid a lot of "why can't I find anything through the OpenAPI GUI?"-misunderstandings.

Reply:
fixed.